vehicles/src/modules/permit/application.controller.ts

```diff
@@ -233,4 +222,50 @@ export class ApplicationController {
     );
     return result;
   }
+
+  @Roles(Role.WRITE_PERMIT)
+  @Delete()
+  @ApiOperation({
+    summary: 'Delete application in progress associated with a company and user(optional)',
+    description:
+      'Allows deletion of one or more applications in progress associated with a given company ID, based on user GUIDs(user GUID needed only for CV clients). ' +
+      'Requires specific user roles or group memberships to execute.' +
+      'Returns a list of deleted applications or throws exceptions for unauthorized access or operational failures.',
+  })
+  async deleteApplications(
+    @Req() request: Request,
+    @Body() deleteApplicationDto: DeleteApplicationDto,
+  ): Promise<ResultDto> {
+    const currentUser = request.user as IUserJWT;
+    const userGuid =
+      UserAuthGroup.CV_CLIENT === currentUser.orbcUserAuthGroup
```

Comment:
Check should include ORGADMIN?

Reply:
> Check should include ORGADMIN?

Thanks for the observation, as ORGADMIN can work on all applications related to their company so userGuid is not needed in case of ORGADMIN login. Hence the userGuid check is only for CVClient.